lib/money-rails/active_record/monetizable.rb

```diff
@@ -85,10 +85,10 @@ def monetize(*fields)
             #
             # To disable validation entirely, use :disable_validation, E.g:
             #   monetize :price_in_a_range_cents, :disable_validation => true
-            if validation_enabled = MoneyRails.include_validations && !options[:disable_validation]
+            if (validation_enabled = MoneyRails.include_validations && !options[:disable_validation])
```

Comment:
I think we should follow the community style guide — https://github.com/bbatsov/ruby-style-guide#no-parens-if

And it seems like this is a default option for RuboCop